src/main/java/jewellery/inventory/service/SystemEventService.java

```diff
@@ -44,6 +45,7 @@ public <T> void logEvent(EventType type, T entity) {
   }
 
   private <U> Object createMap(U entity) {
+    objectMapper.configure(JsonGenerator.Feature.WRITE_BIGDECIMAL_AS_PLAIN, true);
```

Comment:
Should this configuration of the ObjectMapper be done in a @ Configuration class? I see we are only using it here for now and it may turn out to be a overkill. What do you think?

Reply:
It is just for the objectMapper we are using in the tests so maybe its overkill